Classes/ContentObject/BreadcrumbsContentObject.php

```diff
@@ -21,17 +21,12 @@
  */
 class BreadcrumbsContentObject extends AbstractContentObject
 {
-    /**
-     * @var MenuRepository
-     */
-    protected $menuRepository;
 
     /**
      * @param ContentObjectRenderer $cObj
      */
     public function __construct(ContentObjectRenderer $cObj)
```

Comment:
you can drop the Konstruktor

Reply:
Was not sure about this. Should i create a second commit dropping this constructor ?

Reply:
Have removed the constructor.

Reply:
Readded constructure because of moving back to direct menuRepository usage. See review from Benny below.